src/covid_model_seiir_pipeline/forecasting/model.py

```diff
@@ -83,32 +85,96 @@ def __post_init__(self):
 
 class ODERunner:
 
-    def __init__(self, model_specs: SeiirModelSpecs, init_cond: np.ndarray):
-        self.model_specs = model_specs
-        self.init_cond = init_cond
-
-    def get_solution(self, times, beta, theta=None, solver="RK4"):
-        model = CustomizedSEIIR(**asdict(self.model_specs))
-        if solver == "RK4":
-            solver = RK4(model.system, self.model_specs.delta)
+    def __init__(self, solver_name: str, model_specs: SeiirModelSpecs):
```

Comment:
Tried a refactor to handle multiple algorithms.  Threw it away after it didn't work and just reoriented the args a bit.  Nothing fancy happening here.